### ISSUE TITLE
Latest code can't reproduce YOLOv5s on coco

### ISSUE THREAD

**Issue by leedewdew** [ultralytics/yolov5#9547]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Training

### Bug

With latest code pulled Yesterday, I can't reproduce YOLOv5s on coco which is 37.4%, while I get 30.4%.
However, when I train YOLOv5s with the version mentioned in [this wandb workspace](https://wandb.ai/glenn-jocher/YOLOv5_v61_official/runs/zbp9q48q/overview?workspace=), the mAP at the end of each epoch nearly match what the log shows in wandb. But the latest code doesn't.

### Minimal Reproducible Example

python -m torch.distributed.run --master_port 1234 --nproc_per_node 4 train.py \
    --batch 64 \
    --data data/coco.yaml \
    --weights '' \
    --cfg models/yolov5s.yaml \
    --device 0,1,2,3

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@leedewdew related to DDP mAP drop issue https://github.com/ultralytics/yolov5/pull/9545

**Comment by glenn-jocher** [comment]
@leedewdew good news 😃! Your original issue may now be fixed ✅ in PR #9545. To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!

**Comment by leedewdew** [comment]
Thank you for quick response. I'll try.

**Comment by leedewdew** [comment]
I get 36.7% for YOLOv5s. Is that normal?
---------------------------------------
python train.py --epochs 300 --batch 64 --data data/coco.yaml --weights '' --cfg models/yolov5s.yaml --noplots --device 0,1,2,3,4,5,6,7  --project /runs/train --name v5

**Comment by glenn-jocher** [comment]
@leedewdew seems about right. For README results you should train on segmentation labels:

```
bash get_coco.sh --train --val --segments
```
